docs/guides/user-guide/general-faqs.md

```diff
@@ -12,6 +12,16 @@ This page outlines commonly asked questions and answers about StackBlitz.
 
 # General FAQs
 
+### Why do I have to Re-Authorize to access GitHub repositories?
+As of September 27th, 2023 we've begun a gradual migration from GitHub's OAuth app to their GitHub App. We received a lot of feedback from users saying that they didn't like the *all* or nothing approach that GitHub's OAuth app takes so we've started switching over to the GitHub app itself which allows per organization granularity.
```

Comment:
more like per repository granularity, because with Oauth you can do installation into an org, but no to a specific repo